source/unified-test-format/unified-test-format.rst

```diff
@@ -582,6 +583,58 @@ The structure of this object is as follows:
 
   - ``serverApi``: Optional `serverApi`_ object.
 
+.. _entity_encryptedClient:
+
+- ``encryptedClient``: Optional object. Defines a ClientEncryption object.
+
+  The structure of this object is as follows:
+
+  - ``id``: Required string. Unique name for this entity. The YAML file SHOULD
+    define a `node anchor`_ for this field (e.g.
+    ``id: &encryptedClient0 encryptedClient0``).
+
+  - ``clientEncryptionOpts``: Required document. A value corresponding to a
+    `ClientEncryptionOpts
+    <../client-side-encryption/client-side-encryption.rst#clientencryption>`__.
+
+    The structure of this document is as follows:
+
+    - ``keyVaultClient``: Required string. MongoClient entity from which this
+      encryptedClient will be created. The YAML file SHOULD use an `alias
+      node`_ for a client entity's ``id`` field (e.g.
+      ``keyVaultClient: *client0``).
+
+    - ``keyVaultNamespace``: Required string. The database and collection to use
+      as the keyvault collection for this encryptedClient. The namespace takes
```

Comment:
This is typically written as either "key vault" or "keyVault" in camelCase, so I assume it should be two words here.

```suggestion
      as the key vault collection for this encryptedClient. The namespace takes
```